### PR TITLE
Convert Tutorial to use markdown fenced Roc code

### PR DESCRIPTION
**Pull request by lukewilliamboswell**
Using this to test #5051 which should be merged first. I will update this when it is ready for review. 

Converts all the Tutorial to use markdown instead of hand styled html.

### CODE REVIEW

**Comment by lukewilliamboswell** [comment]
Ready for review, however #5080 should be merged first as that includes fixes which add edge cases for this to render everything correctly.

File: www/generate_tutorial/src/input/tutorial.md
Comment (Anton-4) [inline www/generate_tutorial/src/input/tutorial.md:1867]: It feels a bit strange to have a column with all `Yes`, especially because the information in that column is already given in the sentence. Did you have a particular reason to create the `Yes` column?

**Review by Anton-4 — APPROVED** [review]
Excellent work @lukewilliamboswell, this is so much cleaner :)

**Comment by Anton-4** [comment]
I'll approve the CI run once two other required PRs are merged in

File: www/generate_tutorial/src/input/tutorial.md
Comment (lukewilliamboswell) [reply www/generate_tutorial/src/input/tutorial.md:1867]: It was previously a long list of bullet points. It could stay that way, but I thought this looked nicer. Agree it is a bit strange, but also in context it is showing how all of those functions can receive that input, which is probably surprising for someone new to Roc's type system, and conveys the intended message clearly.

**Pull request merged**
